akka-persistence-typed/src/main/scala/akka/persistence/typed/state/internal/Running.scala

```diff
@@ -231,6 +240,24 @@ private[akka] object Running {
       throw new DurableStateStoreException(setup.persistenceId, currentRevision, cause)
     }
 
+    final def onDeleteSuccess(): Behavior[InternalProtocol] = {
+      if (setup.internalLogger.isDebugEnabled) {
+        setup.internalLogger
+          .debug("Received DeleteSuccess response after: {} nanos", System.nanoTime() - persistStartTime)
+      }
+
+      onWriteSuccess(setup.context)
```

Comment:
Those are for telemetry, are you sure we should call them for deletes?

Reply:
There is no instrumentation yet. I think I remove these calls for deletes and add a TODO so that it can be implemented in the way we want when the instrumentation is added. There is also an `onDeleteInitiated`, which I will remove.

I'll leave the onWriteInitiated and onWriteSuccess for the persist case.